pilot/docker/Dockerfile.proxyv2

```diff
@@ -27,6 +41,14 @@ COPY --from=default /etc/iproute2 /etc/iproute2
 
 COPY istio-iptables /usr/local/bin/istio-iptables
 
+# Add CAP_NET_ADMIN and CAP_NET_RAW as Permitted capabilities
+# to the istio-iptables binary.
+COPY --from=libpcapbuilder /sbin/setcap /sbin/
+COPY --from=libpcapbuilder /lib/x86_64-linux-gnu/libcap.so.2 /lib/x86_64-linux-gnu/libcap.so.2.25 /lib/
```

Comment:
this could possibly be avoided based on the above - assuming the setcap is copied over

Reply:
This is in the distroless image, so even if we install it into the base image it will still need to be copied over here.